{"name": "test_password_validation[RightPassword1-aaaaaaaaaa-The passwords you entered did not match, please try again]", "status": "passed", "parameters": [{"name": "password", "value": "'RightPassword1'"}, {"name": "confirm", "value": "'aaaaaaaaaa'"}, {"name": "expected", "value": "'The passwords you entered did not match, please try again'"}], "start": 1707155472419, "stop": 1707155497914, "uuid": "df77defd-5711-49ba-979f-153735c638ff", "historyId": "bc44ccbb220c40cc67fd51149e1163db", "testCaseId": "c3ebc85199741e4ddd50fba973e75269", "fullName": "tests.test_registration.TestRegistration#test_password_validation", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_registration"}, {"name": "subSuite", "value": "TestRegistration"}, {"name": "host", "value": "Alice"}, {"name": "thread", "value": "12564-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_registration"}]}